routers/comments.js

```diff
@@ -0,0 +1,89 @@
+const express = require('express')
+let db = require('../data/db')
+
+const router = express.Router({
+    mergeParams: true
+})
+
+// 	_______CRUD_________
+
+
+//______________________________________________________________
+// get - (http://localhost:4000/api/posts/:id/comments)
+router.get('/', (req, res) => {
+    const id = req.params.id;
+
+    db.findPostComments(id)
+        .then(data => {
+            res.json(data)
+        })
+        .catch(err => {
+            res.status(500).json({
+                error: 'The post with the specified ID does not exist.'
+            })
+        })
+})
+
+
+//______________________________________________________________
+// get (http://localhost:4000/api/posts/:id/comments/:commentId)
+router.get('/:commentId', (req, res) => {
+    db.findCommentById(req.params.id, req.params.commentId)
+        .then(data => {
+            if (data) {
+                res.json(data)
+            } else {
+                res.status(404).json({
+                    message: 'Comment not found'
+                })
+            }
+        })
+        .catch(err => {
+            res.status(500).json({
+                message: 'Could not retrieve comment'
+            })
+        })
+})
+
+//______________________________________________________________
+// POST - (http://localhost:4000/api/posts/:id/comments)
+router.post('/', (req, res) => {
+    const comment = req.body;
+    const id = req.params.id;
+    console.log(comment)
+
+    if (!comment.text) {
+        res.status(400).json({
+            errorMessage: 'Please provide text for the comment.'
+        })
+    } else {
+
+        const newComment = {
+            text: req.body.text,
+            post_id: id
+        }
+
+        db.insertComment(newComment)
+            .then(comment => {
+                if (comment) {
```

Comment:
the specs for adding a comment requires you to return the newly created comment.  Here you are just returning the comment id as insertComment only returns an object with an id